app/controllers/orders/view.js

```diff
@@ -4,6 +4,11 @@ export default Controller.extend({
   actions: {
     downloadInvoice() {
       window.print();
+    },
+    downloadTickets() {
+      return this.get('ajax').request(`${this.get('model.order.ticketsPdfUrl')}`, {
```

Comment:
why are you using this? why not the loader service. Headers are attached automatically attached to this.

Reply:
There was an unexpected error when I used the loader service. 
`    downloadTickets() {
      this.get('loader')
        .post(`${this.get('model.order.ticketsPdfUrl')}`)
        .catch(() => {
          this.get('notify').error(this.get('l10n').t('An unexpected Error occurred'));
        });
    }`

Reply:
There was an unexpected error when I used it. Maybe because the ticket url was stored in its full form? Will see again.

Reply:
The existing code has used the loader service at 100+ instances in the app, and what you did nowhere. There is a strong reason for that. Read what the service does.